docs/reference/mapping/fields/id-field.asciidoc

```diff
@@ -3,10 +3,14 @@
 
 Each document has an `_id` that uniquely identifies it, which is indexed
 so that documents can be looked up either with the <<docs-get,GET API>> or the
-<<query-dsl-ids-query,`ids` query>>.
+<<query-dsl-ids-query,`ids` query>>. The `_id` can either be assigned at
+indexing time, or a unique `_id` can be generated by {es}. This field is not
+configurable.
 
-The value of the `_id` field is accessible in certain queries (`term`,
-`terms`, `match`, `query_string`, `simple_query_string`).
+The value of the `_id` field is only accessible in certain queries (`term`,
+`terms`, `match`, `query_string`, `simple_query_string`), but is restricted
+from use in aggregations, sorting, or scripting. For those use cases the
```

Comment:
I liked the wording 'duplicate the content', I thought it made the recommendation more clear.